src/test/java/unit/Workspace.java

```diff
@@ -136,10 +136,11 @@ void statusDescribeListReflectUpdate() throws IOException {
     // `terra workspace create --format=json --name=$newName --description=$newDescription`
     String newName = "NEW_statusDescribeListReflectUpdate";
     String newDescription = "NEW status describe list reflect update";
+    // verify mixed case enum works
     TestCommand.runCommandExpectSuccess(
         "workspace",
         "update",
-        "--format=json",
+        "--format=jSON",
```

Comment:
Please add a new test for the case insensitive behavior instead of modifying this existing unrelated one. Perhaps new method(s) in the `unit.Config` class, or a new `unit.Syntax` class altogether.

Suggest including tests for options & parameters, lower/upper/mixed case.

Reply:
Done.